File: frontends/py-sh-go/testdata/t64_prefix_suffix.py
# t64_prefix_suffix: prefix & suffix removal
# diagnostics: program prints its result to stdout
x = "hello"
print(x.removeprefix("he"))
print(x.removesuffix("lo"))
print(x.rsplit("l", 1)[1])
print(x[:x.find("l")])

File: frontends/perl-sh-go/testdata/t64_prefix_suffix.pl
# t64_prefix_suffix: prefix & suffix removal
# diagnostics: program prints its result to stdout
my $x = "hello";
(my $a = $x) =~ s/^he//; print "$a\n";
(my $b = $x) =~ s/lo$//; print "$b\n";
(my $c = $x) =~ s/^.*l//; print "$c\n";
(my $d = $x) =~ s/l.*$//; print "$d\n";
